### PR TITLE
add binutils

### PR DESCRIPTION
**Pull request by dvirtz**
Specify library name and version:  **binutils/2.37**

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/binutils/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import os
+
+from conan import ConanFile
+from conan.tools.gnu import Autotools, AutotoolsDeps, AutotoolsToolchain
+from conans import tools
+from glob import glob
+
+required_conan_version = ">=1.44.0"
```
Comment (SSE4) [inline recipes/binutils/all/conanfile.py:8]: we don't run 1.44 in CI yet. that's why `conan export` fails

@jgsogo I guess we need to improve an error message for that particular case

Comment (dvirtz) [reply recipes/binutils/all/conanfile.py]: Thanks @SSE4!

File: recipes/binutils/all/test_package/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from conan import ConanFile
+from conans import tools
+
+
+class TestPackageConan(ConanFile):
+
+    settings = "os"
+
+    # test_type = "explicit"
```
Comment (SSE4) [inline recipes/binutils/all/test_package/conanfile.py:9]: please remove commented code

**Review by prince-chrismc — COMMENTED** [review]
It looks good but I've never used the new toolchain, so I am holding back my ✔️ till someone more experienced looks it over

**Review by jgsogo — APPROVED** [review]
Looking good here, and CI is happy!

These features should be stable enough to start using them, but let's not go crazy with them yet, we don't need to force the migration of all the recipes yet 😄

**Review by lasote — APPROVED** [review]
👏

**Review by prince-chrismc — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Pull request merged**